src/index.ts

```diff
@@ -47,17 +48,21 @@ function loadLocation() { // Inspiration from https://stackoverflow.com/question
                     complexityCircles.checked = true;
                     break;
 
+                case 'autofft':
+                    autoFft = true;
+                    break;
+
                 default:
                     { // no-case-declaration
                         const [k, v] = item.split('=');
                         if (v !== null) { // Restriction to valued keys
                             const w = v && decodeURIComponent(v);
                             switch (k) {
                                 case 'pt':
-                                    { // no-case-declaration
+                                    { // no-case-declaration
                                         const [x, y] = w.split(';');
                                         if (x !== null && y !== null)
-                                            addPoint(Number(x), Number(y), false);
+                                            addPoint(Number(x), Number(y), false);
```

Comment:
You're doing something weird to my line endings again :D

Reply:
Wow, sorry sir!
Guilt goes to copy-pasting between VS and JsFiddle.

Reply:
No need to be sorry! No harm done. I was trying to be funny!